### PR TITLE
*: add codegen for version/version.go

### PR DESCRIPTION
**Pull request by fanminshi**
cc/ @hasbro17

### CODE REVIEW

**Comment by fanminshi** [comment]
Test:
```sh
fanmins-MacBook-Pro:operator-framework fanminshi$ operator-sdk new test-operator
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/cmd/manager/main.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/apis/apis.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/controller/controller.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/Dockerfile
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/build.sh
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/role.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/role_binding.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/operator.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/scripts/codegen/Boilerplate.go.txt
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/version/versions.go
Run dep ensure ...
could not find project Gopkg.toml, use dep init to initiate a manifest
2018/09/20 15:27:00 failed to exec dep []string{"ensure", "-v"}: exit status 1
fanmins-MacBook-Pro:operator-framework fanminshi$ cat test-operator/version/versions.go
package version

var (
	Version = "0.0.1"
)
```

File: commands/operator-sdk/cmd/new.go [hunk 1]
```diff
@@ -265,6 +265,22 @@ func doScaffold() {
...
+	versionDir := filepath.Join(fullProjectPath, "version")
+	if err := os.MkdirAll(versionDir, defaultDirFileMode); err != nil {
+		log.Fatalf("failed to create %v: %v", versionDir, err)
+	}
+
+	// generate version/version.go
+	buf = &bytes.Buffer{}
+	versionGen := scaffold.NewVersionCoden()
+	versionGen.Render(buf)
+	versionPath := filepath.Join(versionDir, "versions.go")
```
Comment (hasbro17) [inline commands/operator-sdk/cmd/new.go]: `versions.go`==>`version.go`

Comment (fanminshi) [reply commands/operator-sdk/cmd/new.go]: fixed.

**Comment by hasbro17** [comment]
LGTM

**Pull request merged**